osmaxx/api_client/conversion_api_client.py

```diff
@@ -85,19 +76,20 @@ def create_job(self, extraction_order, request):
             }
         })
         self.login()
-        response = self.authorized_post(self.conversion_job_url, json_data=request_data)
-        if self.errors:
-            logging.error('API job creation failed.', response)
+        try:
```

Comment:
Even though this doesn't thematically belong to this pull request, this entire code block doesn't work with the new API anymore.